src/app/tree-grid/tree-grid-batch-editing/tree-grid-batch-editing-sample.component.html

```diff
@@ -1,36 +1,39 @@
 <div class="grid__wrapper">
     <app-tree-grid-with-transactions>
-        <igx-tree-grid igxPreventDocumentScroll  #treeGrid [data]="data" primaryKey="ID" foreignKey="ParentID" [rowEditable]="true"
-            [width]="'100%'" [height]="'500px'">
-            <igx-column field="ID" header="ID" dataType="number" width="200"></igx-column>
-            <igx-column field="Age" header="Age" dataType="number" width="120"></igx-column>
-            <igx-column field="Name" header="Full Name" dataType="string" width="240"></igx-column>
-            <igx-column field="Title" header="Title" dataType="string" width="180"></igx-column>
-            <igx-column field="HireDate" header="Hire Date" dataType="date" width="150"></igx-column>
-            <igx-column field="OnPTO" header="On PTO" dataType="boolean" width="80"></igx-column>
-            <igx-column filterable="false" width="150" [editable]="false" >
+        <igx-tree-grid igxPreventDocumentScroll #treeGrid [data]="data" primaryKey="ID" foreignKey="ParentID"
+            [width]="'100%'" [height]="'500px'" [rowEditable]="true">
+            <igx-column filterable="false" width="150" [editable]="false">
```

Comment:
If there's width set the grid looks like this:
![treegrid column](https://user-images.githubusercontent.com/9549439/115855773-51a5b100-a434-11eb-9cfa-e63a432c721e.png)
The DELETE button of the more nested children is not visible.
If the width is removed the button becomes visible even for deeply nested children:
![tree grid nested children](https://user-images.githubusercontent.com/9549439/115855955-84e84000-a434-11eb-8c2f-4a98cbaf3272.png)

Reply:
@simeonoff what you think about this sample. Should we set column width or could we move the delete and add button to the right end side of the grid. Note I am trying to simplify this sample as much as I can and focus on transactions - no templated cell, filtering, sorting or anything not related to the topic.

Reply:
Can you use the actionstrip component to host the row actions?

Reply:
Yes, we can switch to actionstrip, but this will move off the focus of the sample. This is why IMO we should not use actionstrip.